topics/variant-analysis/tutorials/somatic-variant-discovery/tutorial.md

```diff
@@ -0,0 +1,725 @@
+---
+layout: tutorial_hands_on
+
+title: Somatic Variant Discovery from WES Data Using Control-FREEC
+zenodo_link: https://doi.org/10.5281/zenodo.2582555
+questions:
+- What are the specific challenges in locating human Copy Number Variances (hCNVs)?
+- How to preprocess the sequenced reads for hCNVs detection?  
+- How can you detect the hCNVs in/from tumor and normal tissue of the same individual?
+- How can you visualise the hCNVs' findings and compare them for specific regions? 
+objectives:
+- Use Control-Freec for hCNV Identefication in tumor tissue. 
+- Visualise the detected hCNVs in specific chromosomes.
+time_estimation: 3H
+key_points:
+- Follow best practices for read mapping, quality control and mapped reads postprocessing
+  to minimize false-positive hCNVs.
+contributors:
+- Khaled Jum'ah
+- Katarzyna Kamieniecka
+- Wolfgang Maier
+- David Salgado
+- Krzysztof Poterlowicz

```

Comment:
These need to be usernames, not the human names, just as they are in CONTRIBUTORS.yaml :) 

Also, if you like can use our fancy new contributions syntax to more accurately record contributions to a training material, declaring which people should be recorded for authorship or just editing/testing/etc.

```
contributions:
  authorship:
    - shiltemann
  editing:
    - bebatut
    - hexylena
  funding:
    - carpentries
  testing:
    - userX
  ux:
    - userY
  infrastructure:
    - userZ
```

see the [full gtn tutorial schema documentation](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/schemas/tutorial.html) for details on the individual roles.